src/constants.lua

```diff
@@ -1,24 +1,40 @@
 local constants = {}
 
-constants.oneYearSeconds = 60 * 60 * 24 * 365
-constants.thirtyDaysSeconds = 60 * 60 * 24 * 30
+--- @param ARIO number
+--- @return mARIO mARIO the amount of mario for the given ARIO
+function constants.ARIOToMARIO(ARIO)
+	return ARIO * 1000000
```

Comment:
Consider a constant for mARIO_per_ARIO

Reply:
Meh I guess if this function is the one stop shop, then it's also fine.

Reply:
[0968728](https://github.com/ar-io/ar-io-network-process/pull/356/commits/0968728ed472b97ae3c7438b36b1c7eee2f439d5)